packages/database/lib/schema.ts

```diff
@@ -39,8 +39,17 @@ function pgEnum<T extends string>(columnName: string, enumObj: Record<any, T>) {
   return varchar(columnName, {enum: Object.values(enumObj) as NonEmptyList<T>});
 }
 
+export const daos = pgTable('daos', {
+  daoAcct: pubkey('dao_acct').primaryKey(),
```

Comment:
https://github.com/metaDAOproject/futarchy/blob/fbb160788c1a0df54c05de164fc9ab8046026595/programs/autocrat_v0/src/lib.rs#L47

So we use:
- treasuryAccountKey (treasury)
- baseMintKey (token_mint)
- passThreshold (pass_threshold_bps)
- slotsPerProposal (slots_per_proposal)
- (max_observation_change_per_update_lots)
- (base_lot_size)

We'll want to know which program version.. As we do migrate, but would be nice to quickly reference.

Reply:
Should base_lot_size be part of the DAO or part of token metadata we store for the dao?